apollo-adminservice/src/test/resources/application.properties

```diff
@@ -17,7 +17,7 @@ spring.cloud.consul.enabled=false
 spring.cloud.zookeeper.enabled=false
 spring.cloud.discovery.enabled=false
 
-spring.datasource.url = jdbc:h2:mem:~/apolloconfigdb;mode=mysql;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1
+spring.datasource.url = jdbc:h2:mem:~/apolloconfigdb;mode=mysql;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;BUILTIN_ALIAS_OVERRIDE=TRUE
```

Comment:
Is it also necessary to add it to `apollo-configservice/src/test/resources/application.properties`?

Reply:
@nobodyiam Add will be better. Keep consistent